### ISSUE TITLE
Double github repo 

### ISSUE THREAD

**Issue by GitWatin** [Requarks/wiki#2120]
**Question**
Hello, 

By associating WikiJS with my github I have a small problem.
Indeed, when I clone the repository manually using "git clone .... "
It is well imported but not yet visible in the "page" tab.
But once I click on the import button it executes it recreates me the directory in the directory as visible on the picture below foler into principal folder. 

Can you help me? 
Regards

**Host Info (please complete the following information):**
OS: CentOS 8
Wiki.js version: 2.4.107
Database engine: Postgresql
![wikijs](https://user-images.githubusercontent.com/28529151/85959767-9ee4fd80-b99e-11ea-8737-230311d56687.PNG)
 12

**Comment by NGPixel** [comment]
Check your repository folder in the git storage module. You're most likely pointing to the subfolder.

**Comment by GitWatin** [comment]
Hello , 

Thank you for your prompt response.

I don't think I'm pointing at the wrong repertoire.
During configuration I initiated the repo git and got the wiki folder in the /repo_wiki folder.
Then when I clicked import
Then all the content of the wiki repo came into the /repo_wiki folder.

Kind regards 
![repo](https://user-images.githubusercontent.com/28529151/86010773-26248680-ba1c-11ea-9e06-4a87dcd6771a.PNG)
